docs/batch.rst

```diff
@@ -0,0 +1,122 @@
+################
+Batch operations
+################
+
+.. _batch:
+
+POST /batch
+===========
+
+**Requires an FxA OAuth authentication**
+
+The POST body is a mapping, with the following attributes:
+
+- ``requests``: the list of requests
+- ``defaults``: (*optional*) values in common for all requests
```

Comment:
While reading the code (without more information), I found it hard to unsderstand what this was for. We probably should rename this argument to `default_values` or `default_request_values`, that would make it clearer.